src/components/schemas/expense.yaml

```diff
@@ -810,4 +810,155 @@ expense_creation_from_receipt_in:
         This field takes the base64 encoded representation of the file.
       example: JVBERi0xLjQKMSAwIG9iago8PAovVGl0bGUgKP7/KQovQ3JlYX...
     source:
-      $ref: './fields.yaml#/source'
\ No newline at end of file
+      $ref: './fields.yaml#/source'
+
+expense_check_policies_in:
+  type: object
+  additionalProperties: False
+  required:
+    - claim_amount
+    - spent_at
+  properties:
+    id:
+      allOf:
+        - $ref: './fields.yaml#/id_string'
+      description: |
+        Expense id provided by Fyle during the creation of expense.
+    spent_at:
+      $ref: './fields.yaml#/spent_at'
+    source:
+      $ref: './fields.yaml#/source'
+    merchant:
+      $ref: './fields.yaml#/merchant'
+    foreign_currency:
+      $ref: './fields.yaml#/foreign_currency'
+    foreign_amount:
+      allOf:
+        - $ref: './fields.yaml#/foreign_amount'
+      description: |
+        Amount value of the object, in original currency i.e the currency in which the expense was initially created.
+    claim_amount:
+      $ref: './fields.yaml#/claim_amount'
+    purpose:
+      type: string
+      example: Team lunch
+      description: |
+        Purpose of this expense.
+    cost_center_id:
+      allOf:
+        - $ref: './fields.yaml#/fk_integer'
+      description: |
+        This represents the cost_center's id given by Fyle during creation of cost center
+        for which this expense is created.
+      nullable: true
+      example: 1049
+    category_id:
+      allOf:
+        - $ref: './fields.yaml#/fk_integer'
+      description: |
+        This represents the category's id given by Fyle during creation of category
+        for which this expense is created.
+      nullable: true
+      example: 49058
+    project_id:
+      allOf:
+        - $ref: './fields.yaml#/fk_integer'
+      description: |
+        This represents the project's id given by Fyle during creation of project
+        for which this expense is created.
+      nullable: true
+      example: 92895
+    source_account_id:
+      allOf:
+        - $ref: './fields.yaml#/fk_string'
+      description: |
+        This represents the source account's id given by Fyle during creation of account type
+        of user for which this expense is created.
+      example: acwbl222wlg
+    tax_amount:
+      type: number
+      nullable: true
+      description: Tax amount as setup by your organization.
```

Comment:
If you are not changing the description, please reference to it.

Reply:
This couldn't be referenced. Defined everywhere explicitly inside objects